plugins/hid/hidplugin.cpp

```diff
@@ -88,8 +87,7 @@ void HIDPlugin::closeInput(quint32 input, quint32 universe)
     {
         removeFromMap(input, universe, Input);
         dev->closeInput();
-        disconnect(dev, SIGNAL(valueChanged(quint32,quint32,quint32,uchar)),
-                   this, SIGNAL(valueChanged(quint32,quint32,quint32,uchar)));
+        disconnect(dev, SIGNAL(valueChanged(quint32,quint32,quint32,uchar)), this, SIGNAL(valueChanged(quint32,quint32,quint32,uchar))); // TODO: The slot has one more parameter than the signal. We can connect() using a lambda, but can not easily disconnect.
```

Comment:
This one is tricky... We could `connect()` using a lambda (same reason as https://github.com/mcallegari/qlcplus/pull/1732/files#r2054325313). But `disconnect()`ing using a lambda is not possible of course. [There is a way](https://stackoverflow.com/a/14829520) but I think that would make things only less intuitive. Maybe continue to use the type-unsafe way here?